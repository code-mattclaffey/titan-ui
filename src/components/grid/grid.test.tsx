import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Grid, GridItem } from '.';

afterEach(() => {
    cleanup();
});

describe('<Grid />', () => {
    it('should match snapshot', () => {
        const { container } = render(
            <Grid>
                <GridItem
                    columns={{
                        xs: '1',
                        sm: '3',
                        md: '6',
                        lg: '9',
                        xl: '12',
                    }}
                >
                    <p>Some text</p>
                </GridItem>
                <GridItem
                    columns={{
                        xs: '6',
                        lg: '9',
                        xl: '12',
                    }}
                >
                    <p>Some text</p>
                </GridItem>
            </Grid>,
        );

        expect(container).toMatchSnapshot();
    });
});
