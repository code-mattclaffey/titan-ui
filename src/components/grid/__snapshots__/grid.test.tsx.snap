// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Grid /> should match snapshot 1`] = `
<div>
  <div
    class="o-grid"
  >
    <div
      class="o-grid__item o-grid__item--1/12 o-grid__item--3/12@sm o-grid__item--6/12@md o-grid__item--9/12@lg o-grid__item--12/12@xl"
    >
      <p>
        Some text
      </p>
    </div>
    <div
      class="o-grid__item o-grid__item--6/12 o-grid__item--9/12@lg o-grid__item--12/12@xl"
    >
      <p>
        Some text
      </p>
    </div>
  </div>
</div>
`;
